### ISSUE TITLE
using the source code to list directories on private bucket

### ISSUE THREAD

**Issue by mohdmulla** [rufuspollock/s3-bucket-listing#94]
I used the source code and it is working perfectly for list public bucket contents 

Is it possible to list objects/directories that are in a private bucket using the source code you provided? and what changes should I make to achieve my goal

**Comment by munntjlx** [comment]
I would be interested in this too!

**Comment by prathameshd9** [comment]
yeah this would be a great functionality to have..especially if someone is displaying the site behind a private VPN on a private bucket

**Comment by RayWelker** [comment]
I'd also be interested in this.

**Comment by jon-f-novabackup** [comment]
I second this request, is there an alternative that can do directory listings for private buckets, such as those using signed URLs?

**Comment by Dhyanesh97** [comment]
I have same requirement.

**Comment by rufuspollock** [comment]
Maybe we can start a bounty for this.
